torch/lib/THNN/generic/HardTanh.c

```diff
@@ -26,6 +26,7 @@ void THNN_(HardTanh_updateOutput)(
         else if (*input_data > max_val)
           *input_data = max_val;
       );
+    else
```

Comment:
wow, nice catch. Can you add brackets around both branches to make it more explicit? Thanks!

Reply:
There are two styles of brackets using in pytorch: (I choose the first)
1. else {
2. else
    {